device-types/Cisco/WS-C3650-24PDM-L.yaml

```diff
@@ -0,0 +1,122 @@
+---
+manufacturer: Cisco
+model: Catalyst 3650-24PDM-L
+slug: cisco-ws-c3650-24pdm-l
+part_number: WS-C3650-24PDM-L
+is_full_depth: false
+u_height: 1
```

Comment:
Please add `module-bays` with the 2 stacking ports

Reply:
All C3650, C3750X, and C3850 models support stacking cables, according to the data sheets, and yet none of these pre-existing device type files make any mention of stacking modules. Such a change is beyond the scope of this PR / issue

I think a separate issue / PR should be opened to address that.

Reply:
Stack ports are fixed ports, they shouldn't be modular

Reply:
Dan and I had a conversation, I should actually change how I phrase this in the future. If the stacking ports are modular, then they should be in module-bays.. Otherwise they should be in interfaces

Thanks for the catch Dan

Reply:
It does look like the "L" versions are modular, but the 3750X, and straight 3850's are non-modular.  Not sure about the 3650's but I imagine they are non-modular as well.